packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -55,7 +55,7 @@ export const TooltipTrigger = (
     ref: ForwardedRef<HTMLButtonElement>,
 ) => {
     return (
-        <RadixTooltip.Trigger data-test-id={dataTestId} asChild ref={ref}>
+        <RadixTooltip.Trigger data-test-id={dataTestId} ref={ref}>
```

Comment:
I haven't seen any side effects on this change but we can expose the `asChild` prop if we want to be on the safe side